ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+        tableView.dataSource = self
+
+        isTableViewLoaded = true
+        return tableView
+    }()
+
+    // MARK: - Command tap handling
+
+    @objc private func handleHeroCommandTap(_ sender: UITapGestureRecognizer) {
+        guard let tabBarItemView = sender.view as? TabBarItemView, let binding = viewToBindingMap[tabBarItemView] else {
+            return
+        }
+
+        switch binding.item.commandType {
+        case .toggle:
+            tabBarItemView.isSelected.toggle()
+            binding.item.isOn = tabBarItemView.isSelected
+            fallthrough
+        case .simple:
+            binding.item.action(binding.item)
+        }
+    }
+
+    @objc private func handleMoreButtonTap(_ sender: UITapGestureRecognizer) {
+        let popoverContentVC = UIViewController()
+        popoverContentVC.view.addSubview(tableView)
+        popoverContentVC.modalPresentationStyle = .popover
+        popoverContentVC.popoverPresentationController?.sourceView = sender.view
+
+        NSLayoutConstraint.activate([
+            popoverContentVC.view.leadingAnchor.constraint(equalTo: tableView.leadingAnchor),
+            popoverContentVC.view.trailingAnchor.constraint(equalTo: tableView.trailingAnchor),
+            popoverContentVC.view.topAnchor.constraint(equalTo: tableView.topAnchor),
+            popoverContentVC.view.bottomAnchor.constraint(equalTo: tableView.bottomAnchor)
+        ])
+
+        present(popoverContentVC, animated: true)
+    }
+
+    // MARK: - Item <-> View Binding
+
+    private func addBinding(_ binding: ItemBinding) {
+        itemToBindingMap[binding.item] = binding
+        viewToBindingMap[binding.view] = binding
+    }
+
+    private func removeBinding(_ binding: ItemBinding) {
+        itemToBindingMap.removeValue(forKey: binding.item)
+        viewToBindingMap.removeValue(forKey: binding.view)
+    }
+
+    private func clearAllItemViews(in location: ItemLocation) {
+        switch location {
+        case .heroSet:
+            heroItems.forEach {
+                if let binding = itemToBindingMap[$0] {
+                    removeBinding(binding)
+                }
+            }
+            heroCommandStack.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        case .list:
+            expandedListSections.forEach {
+                $0.items.forEach {
+                    if let binding = itemToBindingMap[$0] {
+                        removeBinding(binding)
+                    }
+                }
+            }
+        }
+    }
+
+    private func createAndBindHeroCommandView(with item: CommandingItem) -> UIView {
+        let tabItem = TabBarItem(title: item.title, image: item.image, selectedImage: item.selectedImage)
+        let itemView = TabBarItemView(item: tabItem, showsTitle: true)
+        itemView.alwaysShowTitleBelowImage = true
+        itemView.numberOfTitleLines = 1
+        itemView.isSelected = item.isOn
+        itemView.accessibilityTraits.insert(.button)
```

Comment:
also need to support larger content viewer now or later for larger text support

Reply:
Done.